eberron.md

```diff
@@ -113,22 +123,21 @@ When cast at 6th level or above, the weapon gains Reach 2.
 - **Range:** self
 - **Duration:** 10 minutes.
 
-Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. The spell ends if they regain enough wounds to regain consciousness.
+Cast against 2 terrible dice, targeting an unconscious creature. The creature rises in a stupor and are no longer incapacitated. While in this state they keep their temporary wounds and continue to roll death saving throws. They cannot take any action except movement and attacks, and any turn they do not take damage, are attacked, or attack another creature they automatically fail their death saving throw. While under this effect they ignore the first wound of damage they take from attacks each round. The spell ends if they regain enough wounds to regain consciousness.
 
 For each level cast above 3rd, add an enhanced die.
 
-Reach increases by 1 and you add a white die every level.
-
 #### Ritual of Healing Blood
 *3rd Level*
 
 - **Casting Time:** 9 AP
 - **Range:** 3
 - **Duration:** instant.
+- **Damage Type:** death
 
-Pull the life force from a creature to heal another. Target up to 3 injured and living creatures. Make a spellcasting roll adding a bad die for each creature targeted past the first. Creatures make a stamina save against this attack, taking 2 wound dice in death damage on fail. A target creature in range heals for 1 + half the damage dealt.
+Pull the life force from a creature to heal another. Target up to 5 injured and living creatures. Make a spellcasting roll adding a terrible die for each creature targeted past the first. Creatures make a stamina save against this attack, taking 2 wound dice in death damage on fail. A target creature in range heals for 1 + half the damage dealt.
```

Comment:
you can keep it at 3 if you want. Most of my multi target spells go up to 5, adding negatives per creature as a way of indirectly saying you can target an extra person per level and rewarding invested casters. Highly invested casters can make full use of the spell without needing to upcast it compared to people that just dip into blood magic. It's also why spells like these usually add enhanced dice as opposed to wounds or something else.